src/components/MessageBox.js

```diff
@@ -0,0 +1,96 @@
+const React = require('react');
+
+const StyleConstants = require('../constants/Style');
+const Icon = require('../components/Icon');
+
+const MessageBox = React.createClass({
+  propTypes: {
+    children: React.PropTypes.node,
+    color: React.PropTypes.string,
+    expandable: React.PropTypes.bool,
```

Comment:
Good point about having to use callbacks, so never mind about the `isOpen` prop idea and because there isn't a use case for it yet.